### DISCUSSION TITLE
Output IR commands without the LED modulation

### DISCUSSION THREAD

**Discussion by pxpto** [arendst/Tasmota#10808] [Q&A] [answered]
Hi,

I would like to use Tasmota to send IR commands to a device, but electrically - not optically. I mean, instead of driving a LED with the modulated 38 kHz signal, I would like Tasmota to give me the un-modulated TTL levels of the command so I can apply it directly to the IR receiver chip inside the device.

Is there a way to achieve this with Tasmota?

Thank you in advance!

**Comment by barbudor** [comment, marked as answer]
You need to compile your own firmware with 
`#define IR_SEND_USE_MODULATION    false`
in your `user_config_override.h`file

For compiling your own firmware read : https://tasmota.github.io/docs/Compile-your-build/

**Reply by pxpto** [reply]
Thanks! I'll try that. It would be nice to be able to change it in runtime with a command.

**Reply by pxpto** [reply]
Thank you @barbudor. It worked perfectly. I also needed IR_SEND_INVERTED and there it was! These two settings were merged a few days after the latest stable build. They came just in time! :)
